tests/common.go

```diff
@@ -5274,8 +5274,8 @@ func IsNFSSubPathEmpty(subPath string) (bool, error) {
 	creds := GetNfsInfoFromEnv()
 	mountDir := fmt.Sprintf("/tmp/nfsMount" + RandomString(4))
 
-	// Mount the NFS share to the master node.
-	masterNode := node.GetMasterNodes()[0]
+	// Mount the NFS share to the worker node.
+	masterNode := node.GetWorkerNodes()[0]
```

Comment:
Let's add a todo here for supporting clouds.

Reply:
Done